tests/test_datetime/test_extract_date_features.py

```diff
@@ -43,13 +43,19 @@ def test_extract_date_features(df_vartypes2):
             }
         )
     )
+    dates_nan = pd.DataFrame({"dates_na": ["Feb-2010", np.nan, "Jun-1922", np.nan]})
 
     # check exceptions upon class instantiation
```

Comment:
In case you're wondering what's the big fuss about it well tests were thorough and that but I came across a huge error i.e. I had more than one error assertion per pytest.raises with block so basically only the first one per block was being checked.